docs/licensing.md

```diff
@@ -17,7 +20,22 @@ The current limitations are applied to the community version:
     You need to pass in a setting to Qlik Associative Engine to acknowledge that
     you accept the EULA. Example: `docker run qlikcore/engine:<version> -S AcceptEULA=yes`
 
-## Licensed
+## 2. Evaluation Trial License
+
+To better evaluate Qlik Core without the barriers in the developer edition, register to obtain a
+free 90-day license key.
+
+With this evaluation trial license, you can:
+
+* Avoid having the Qlik Associative Engine expire 30 days after the version has been released
+* Better evaluate scalability and load testing abilities with no limit on concurrent Qlik Associative Engine sessions
+
+!!! Note "Getting a license"
```

Comment:
Isn't it easier to add this notice under the "Licensed" section to avoid talking about three options when there's really 2,5? :) "Developer" and "Licensed", where "Licensed" may be used using a trial license for 90 days?

Reply:
Did some rephrasing but we keep 3 options for now.